addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick_multi.lua

```diff
@@ -10,24 +10,24 @@ TOOL.Command    = nil
 TOOL.ConfigName = ""
 
 if CLIENT then
-  language.Add( "tool."..gsToolModeOP..".name"      , "Joystick Multi Tool (Wire)" )
-  language.Add( "tool."..gsToolModeOP..".desc"      , "Spawns a Joystick Module interface chip for use with the wire system." )
-  language.Add( "tool."..gsToolModeOP..".0"         , "Primary: Create/Update Joystick   Secondary: Copy Settings    Reload: Link to pod" )
-  language.Add( "tool."..gsToolModeOP..".1"         , "Now select the pod to link to, or anything other than a pod to revert.")
-  language.Add( "tool."..gsToolModeOP..".uid"       , "Unique identifier. No spaces, alphanumeric, 17 charater limit" )
-  language.Add( "tool."..gsToolModeOP..".uid_con"   , "UID")
-  language.Add( "tool."..gsToolModeOP..".bdescr"    , "Write some input description here. Maximum 20 characters. For example `Steering`" )
-  language.Add( "tool."..gsToolModeOP..".bdescr_con", "Description")
-  language.Add( "tool."..gsToolModeOP..".lcontr"    , "This labels the given set of input control configuration settings" )
-  language.Add( "tool."..gsToolModeOP..".lcontr_con", "Control configuration:" )
-  language.Add( "tool."..gsToolModeOP..".maxon"     , "Maximum output value when analogue or ON value when digital" )
-  language.Add( "tool."..gsToolModeOP..".maxon_con" , "Maximum / On")
-  language.Add( "tool."..gsToolModeOP..".minoff"    , "Minimum output value when analogue or OFF value when digital" )
-  language.Add( "tool."..gsToolModeOP..".minoff_con", "Minimum / Off" )
-  language.Add( "tool."..gsToolModeOP..".analog"    , "Enable this when your source is analogue input" )
-  language.Add( "tool."..gsToolModeOP..".analog_con", "Analog input" )
-  language.Add( "tool."..gsToolModeOP..".config"    , "Click this button to open joystick configuration. You can also righ click on the world" )
-  language.Add( "tool."..gsToolModeOP..".config_con", "Joystick Configuration" )
+  language.Add( "tool."..gsToolModeOP..".name"           , "Joystick Multi Tool (Wire)" )
+  language.Add( "tool."..gsToolModeOP..".desc"           , "Spawns a Joystick Module interface chip for use with the wire system." )
+  language.Add( "tool."..gsToolModeOP..".0"              , "Primary: Create/Update Joystick   Secondary: Copy Settings    Reload: Link to pod" )
+  language.Add( "tool."..gsToolModeOP..".1"              , "Now select the pod to link to, or anything other than a pod to revert.")
+  language.Add( "tool."..gsToolModeOP..".uid"            , "Unique identifier. No spaces, alphanumeric, 17 charater limit!" )
+  language.Add( "tool."..gsToolModeOP..".uid_con"        , "UID")
+  language.Add( "tool."..gsToolModeOP..".description"    , "Write some input description here. Maximum 20 characters! For example `Steering`" )
```

Comment:
Yes... In fact this is hardcoded also along the file. At Least utilized language.Add